### PR TITLE
Added Generic to Class, Added atribute list, Added list method

### PR DESCRIPTION
**Pull request by AlejandroWoloch**
So, I added a list atribute and the list method to de ABML, let me know if you wanted this way, or do you want to make T extend from somthing?

### CODE REVIEW

**Comment by ducampcarlos** [comment]
Looks good to me

File: src/ABML.java [hunk 1]
```diff
@@ -1,4 +1,18 @@
+import java.util.ArrayList;
+import java.util.List;
 
-public class ABML {
+public class ABML<T> {
 
+	    List<T> list= new ArrayList<T>(); //Se instancia en el constructor
```
Comment (rightonhana) [inline src/ABML.java]: Remove the instance of ArrayList (or set it to null)

**Comment by ducampcarlos** [comment]
Lgtm, very star ass

**Pull request merged**